docs/tutorials/onnx/export_mxnet_to_onnx.md

```diff
@@ -0,0 +1,100 @@
+
+# Exporting MXNet model to ONNX format
+
+[Open Neural Network Exchange (ONNX)](https://github.com/onnx/onnx) provides an open source format for AI models. It defines an extensible computation graph model, as well as definitions of built-in operators and standard data types.
+
+In this tutorial, we will show how you can save MXNet models to ONNX format.
+
+Current MXNet-ONNX import and export operator support and coverage can be found [here](https://cwiki.apache.org/confluence/display/MXNET/ONNX+Operator+Coverage).
+
+In this tutorial we will:
+    
+- learn how to use MXNet to ONNX exporter on pretrained models
+
+## Pre-requisite
+
+To run the tutorial you will need to have installed the following python modules:
+- [MXNet == 1.3.0](http://mxnet.incubator.apache.org/install/index.html)
+- [onnx](https://github.com/onnx/onnx) v1.2.1 (follow the install guide)
+MXNet ONNX importer and exporter follows version 7 of ONNX operator set.
```

Comment:
I'm not sure what this means. I see you want ONNX v1.2.1, but there's yet another version to know about within ONNX? Would I get v1.2.1 then have to download an operator set that is version 7? Maybe this is TMI?

Reply:
v1.2.1 comes with opset 7 which we currently support. Made it more clear